clash-starters/clash-example-project/LICENSE

```diff
@@ -0,0 +1,6 @@
+Copyright (c) 2020, YOUR COMPANY OR NAME
```

Comment:
Change 2020 to YEAR, or be doomed to having to remember to change it every year.
I would also change "YOUR COMPANY OR NAME" to just "AUTHORS", but that's just preference